internal/cmd/asyncapi/account_details.go

```diff
@@ -74,24 +74,38 @@ func (d *accountDetails) getSchemaDetails() error {
 	if err != nil {
 		return err
 	}
-	var unmarshalledSchema map[string]any
+	d.channelDetails.schema = &schema
+	unmarshalledSchema := make(map[string]any)
 	if schema.SchemaType == "" {
-		d.channelDetails.contentType = "application/avro"
-	} else if schema.SchemaType == "JSON" {
+		schema.SchemaType = "AVRO"
+	}
+	switch schema.SchemaType {
+	case "JSON":
 		d.channelDetails.contentType = "application/json"
-	} else if schema.SchemaType == "PROTOBUF" {
+		break
+	case "AVRO":
+		d.channelDetails.contentType = "application/avro"
+		break
+	case "PROTOBUF":
 		log.CliLogger.Warn("Protobuf not supported.")
 		d.channelDetails.contentType = "PROTOBUF"
```

Comment:
Why do we set the `contentType` if it's not supported?

Reply:
It has no use right now.